examples/teleport-usage/Dockerfile

```
@@ -3,6 +3,7 @@ WORKDIR /app
 COPY go.mod go.sum main.go /app/
 RUN go build -o teleport_usage
 FROM debian:buster-slim
+RUN apt update && apt install -y ca-certificates && rm -rf /var/lib/apt/lists/*
```

```suggestion
RUN apt-get -y update && \
   apt-get install -y ca-certificates && \
   apt-get -y clean && \
   rm -rf /var/lib/apt/lists/*
```

Let's use `apt-get` because I'm pretty sure `apt` is supposed to be for interactive use while `apt-get` for non-interactive use.